index.html

```diff
@@ -250,6 +250,17 @@ <h3 class='txt-h4'><span>Routable Points<span><span class='ml12 txt-s color-red-
                             </div>
                         </div>
 
+                        <!--local search Toggle-->
+                        <div class='col col--12'>
+                            <div class="col col--12 py12">
+                                <label class='switch-container fl pr12'>
+                                    <input v-model="cnf.localsearch" type='checkbox'/>
+                                    <div class='switch'></div>
+                                </label>
+                                <h3 class='txt-h4'><span>Local Search<span><span class='ml12 txt-s color-red-light color-red-dark-on-hover'>BETA<span></h3>
```

Comment:
Should we hide this for non-mapbox users for the time being? Or should we add a popup for more information about the feature? Right now this bucks the trend of current settings as it is not a query param but a different access key.

We definitely want this, just don't want to confuse users with how this works. IE: You can't access this without first contacting us.

Reply:
Good point @ingalls. Per voice with @karenell, I added a tooltip that adds additional information to the toggle.